src/replica/replica_stub.cpp

```diff
@@ -1925,6 +1953,28 @@ void replica_stub::open_replica(const app_info &app,
                req ? "with" : "without",
                dir.c_str());
         rep = replica::load(this, dir.c_str());
+
+        // if load data failed, re-open the `*.ori` folder which is the origin replica dir of disk
+        // migration
+        if (rep == nullptr) {
+            std::string origin_dir = get_replica_dir(
+                fmt::format("{}{}", app.app_type, replica_disk_migrator::kReplicaDirOriginSuffix)
+                    .c_str(),
+                id,
+                false);
+            if (!origin_dir.empty()) {
+                ddebug_f("start revert and load disk migration origin replica data({})",
+                         origin_dir);
+                dsn::utils::filesystem::rename_path(dir, fmt::format("{}.{}", dir, "gar"));
+
+                std::string revert_dir = origin_dir;
+                boost::replace_first(revert_dir, ".disk.balance.ori", "");
```

Comment:
what about sub_str?

Reply:
I modify `origin_tmp_dir` and  `origin_dir`